src/pages/LikedPage/LikePage.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import "./LikePage.css";
+import axios from "axios";
+import {useEffect} from "react";
+import VideoCards from "../../components/UI/Cards/VideoCards/VideoCards";
+import WatchLaterVideoCards from "../../components/UI/Cards/WatchLaterVideoCards/WatchLaterVideoCards";
+import {useWatchlater} from "../../context/WatchLaterContext";
+import {
+    HolderImg1,
+    HolderImg2,
+    HolderImg3,
+    HolderImg4,
+    HolderImg5,
+    HolderImg6,
+    HolderImg7,
+    HolderImg8,
+    HolderImg9
+} from "../../assets/Holders/holder";
+import {VAR_ENCODE_TOKEN} from "../../utils/Route";
+import { useLikes } from '../../context/LikesContext';
+import LikedVideoCards from '../../components/UI/Cards/LikedVideoCards/LikedVideoCards';
+
+const LikePage = () => {
+  const { likesContextArray, setLikesContextArray } = useLikes();
+  useEffect(() => {
+		try {
+			(async () => {
+				var res = await axios.get("/api/user/likes", {
+					headers: {
+						authorization: localStorage.getItem(VAR_ENCODE_TOKEN),
+					},
+				});
+				console.log(res);
```

Comment:
Suggestion: You can remove these console.log if not required...

Reply:
done